frontend/src/pages/meditation/components/meditation-list/meditation-entry/styles.module.scss

```diff
@@ -31,9 +31,16 @@
 }
 
 .title {
+  display: -webkit-box;
```

Comment:
-webkit-box is not supported in firefox. Check this [link](https://caniuse.com/?search=webkit-box)

Reply:
@fshabanov I think you confused it with -webkit-box-reflect?

Also, it was added for titles in this [issue](https://github.com/orgs/BinaryStudioAcademy/projects/12/views/1?filterQuery=&sliceBy%5Bvalue%5D=OlesiaTr&pane=issue&itemId=38782842) and its done being tested, I wasn't contacted by QA team about it's not working on Firefox either.

![webkit-box](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/104642736/a34e1788-cfbf-4b26-aa8f-845e710f4cb4)